src/hotspot/cpu/x86/vm_version_x86.cpp

```diff
@@ -2863,185 +2886,210 @@ int64_t VM_Version::maximum_qualified_cpu_frequency(void) {
   return _max_qualified_cpu_frequency;
 }
 
-uint64_t VM_Version::CpuidInfo::feature_flags() const {
-  uint64_t result = 0;
```

Comment:
It's unfortunate you migrated away from operating on a local copy. Why don't you declare a local copy (`VM_Version result`) and migrate bit manipulation to bit field accessors on it? `VM_Version::CpuidInfo::feature_flags()` can still return it by value (once you get rid of heap memory allocation, copying becomes trivial).

Reply:
New implimentation directly modify the feature vector bits though macros.

Reply:
I prefer explicit accessor calls on corresponding instance fields. 

It's confusing to see `VM_Version::CpuidInfo::feature_flags()` implicitly modifying `_dynamic_features_vector` through macros.

Reply:
VM_Version::CpuidInfo::feature_flags() is local to x86 targets, how about changing its name to VM_Version::CpuidInfo::install_feature_flags() and use macros ?

Reply:
I have changed this local rountine name to install_feature_flags to confirm to its semantics